crates/gpui/src/platform/blade/blade_renderer.rs

```diff
@@ -128,7 +129,7 @@ struct BladePipelines {
 }
 
 impl BladePipelines {
-    fn new(gpu: &gpu::Context, surface_info: gpu::SurfaceInfo) -> Self {
+    fn new(gpu: &gpu::Context, surface_info: gpu::SurfaceInfo, sample_count: u32) -> Self {
```

Comment:
the argument should say explicitly that it's only for the path rasterization, e.g. `path_sample_count`